store/errors.go

```diff
@@ -183,28 +191,38 @@ func (e SnapActionError) Error() string {
 			header = "cannot install or download:"
 		}
 	}
-	es := []string{header}
 
-	for name, e := range e.Refresh {
-		es = append(es, fmt.Sprintf("snap %q: %v", name, e))
-	}
+	// reverse the maps, as the common case is that they're all the same
+	reversed := map[string][]string{}
```

Comment:
(nitpick) maybe call it "errToSnaps" or something which makes the content more obvious?